demos/toolbar/index.html

```diff
@@ -87,10 +89,21 @@
           <header class="mdc-toolbar">
             <div class="mdc-toolbar__row">
               <section class="mdc-toolbar__section mdc-toolbar__section--align-start">
-                <span class="catalog-back">
-                  <a href="#"><i class="material-icons">menu</i></a>
+                <span class="mdc-toolbar__icon--menu">
```

Comment:
The below lines need also be changed to have `mdc-toolbar__icon` directly applied to `<a>` tag :)